src/caveexpress/server/entities/npcs/NPCFriendly.cpp

```diff
@@ -82,9 +82,9 @@ bool NPCFriendly::triggerTargetCaveAnnouncement (const b2Vec2& playerPos)
 		return false;
 	}
 	if (_triggerMovement == 0) {
-		_triggerMovement = _time + 600;
-		GameEvent.announceTargetCave(getVisMask(), *this, 1200);
-		return false;
+		_triggerMovement = _time + 200;  // 600
+		GameEvent.announceTargetCave(getVisMask(), *this, 1400);  // 1200
+		return true;  // false;
```

Comment:
wouldn't this start the movement right is in the same moment the announcement was made?

Reply:
Yes, kind of fast. I think with so many transfers it's a wasted player time to wait. Good for beginners.
Users can either se the announcement while npc walks or on HUD (top).
Now gameplay focues on flying.